public/css/common.less

```diff
@@ -186,10 +186,17 @@ div.show-more {
   padding: .5em;
 }
 
-.object-detail .item-list .list-item:last-of-type .caption {
-  min-height: 1.5em;
-  max-height: 3em;
-  height: auto;
+.object-detail .item-list {
+  &.action-list .list-item {
+    margin: 0 -1em;
+    padding: 0 1em;
```

Comment:
Please use `margin-left` and `margin-right` as well as `padding-left` and `padding-right`.

It's the horizontal margin/padding only that is of interest here, it shouldn't have an effect on the vertical one if not necessary.